docs/source/gcp/examples/pubsub-snapshots.rst

```diff
@@ -0,0 +1,28 @@
+Pub/Sub - Find Snapshots Referring to Deleted Topics
+====================================================
+
+In Cloud Pub/Sub, the snapshot feature allows users to capture the message acknowledgment state of a subscription to a topic. Once a snapshot is created, it retains all messages that were unacknowledged in the source subscription (at the time of the snapshot's creation).
+
+Some Pub/Sub snapshots may refer to deleted topics (GCP uses special ``_deleted-topic_`` name for them). Although such snapshots are deleted in a week upon creation, in order to enforce security Custodian can filter them earlier, thus decreasing time-to-live of unused sensitive data.
```

Comment:
let's reword this entire paragraph, its not nesc a security thing, and we shouldn't assume as such in the docs when presenting generic corner case examples.

`All pubsub snapshots are deleted in a week or less of their creation. For some use cases it maybe useful to delete them earlier.`

Reply:
Thanks, I've generalized the title and description